docs/core/compatibility/wcf-client/6.0/duplex-synchronization-context.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Breaking change: Duplex contract captures synchronization context"
+description: Learn about the breaking change in WCF Client 6.0 where the duplex contract now captures synchronization context when it's opened.
+ms.date: 04/06/2023
+---
+# Duplex contract captures synchronization context
+
+The duplex contract now captures the synchronization context when it's opened and dispatches callbacks to the captured context.
+
+## Previous behavior
+
+In previous versions, the duplex contract failed to capture synchronization context when the channel factory was opened.
```

Comment:
In previous versions, DuplexChannelFactory failed to capture the current synchronization context when the channel factory was opened.